include/vrv/bboxdevicecontext.h

```diff
@@ -80,8 +80,8 @@ class BBoxDeviceContext : public DeviceContext {
     virtual void DrawRectangle(int x, int y, int width, int height);
     virtual void DrawRotatedText(const std::string &text, int x, int y, double angle);
     virtual void DrawRoundedRectangle(int x, int y, int width, int height, int radius);
-    virtual void DrawText(
```

Comment:
Is it worth mentioning this in the PR description? For me it's not clear why this function is being changed from the description

Reply:
Ah yeah good point. It adds the option to set the height and width of the text manually rather than using `TextExtend`. If they're unset then `TextExtend` is used instead. This is for drawing the `syl` bounding boxes from the `zone` elements.